index.html

```diff
@@ -162,62 +170,52 @@
                     <div class="col-xs-12 col-sm-6 col-md-4 col-lg-3">
                         <div class="card stu-card">
                             <div class="avatar">
-                                {% if student.image %}
```

Comment:
You are not allowed to add your name using `index.html`.
Please go to `_data/students.yml` and add your details at the last or somewhere in the last, and it would automatically fetch your name in index.html.
Please Refer to **README.md** in repo for more information.